istioctl/cmd/istioctl/convert/testdata/v1alpha1/destination-rule-helloworld.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: config.istio.io/v1beta1
+kind: DestinationPolicy
+metadata:
+  name: helloworld-circuit-breaker
+spec:
+  destination:
+    name: helloworld-service
+    labels:
+      version: v1
```

Comment:
label mismatch? here the label is `version: v1`, whereas in the virtual service it routes to `version: 1.0` or `version: 2.0`